assets/js/index.js

```diff
@@ -2,38 +2,54 @@ const addTaskIcon = document.getElementById("add-task-icon");
 const addTaskInput = document.getElementById("add-task-input");
 const errorMessage = document.getElementById("error-message");
 const backendErrorMessage = document.getElementById("backend-error-message");
+const todoList = document.getElementById("todo-list");
+const filterItemAll = document.getElementById("filter-item-all");
+const filterItemUncompleted = document.getElementById(
+  "filter-item-uncompleted"
+);
+const filterItemCompleted = document.getElementById("filter-item-completed");
+const url = "http://localhost:3030";
 
 async function postTodo(todoText) {
-  const url = "http://localhost:3030/todos";
-
   try {
-    const response = await fetch(url, {
+    const response = await fetch(`${url}/todos`, {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
       body: JSON.stringify({ text: todoText }),
     });
-    const {error} = await response.json();
+    const { error, data } = await response.json();
```

Comment:
If you are not using data. Please, remove it.

Reply:
Done.

Reply:
Done.